service/src/main/java/org/mskcc/cmo/metadb/service/impl/MessageHandlingServiceImpl.java

```diff
@@ -230,20 +230,10 @@ public void run() {
                         MetadbRequest existingRequest =
                                 requestService.getMetadbRequestById(requestMetadata.getIgoRequestId());
                         if (existingRequest == null) {
-                            LOG.info("Request does not already exist - saving to database: "
```

Comment:
Removed this because the request metadata we receive will not have any samples. It wouldn't make sense to persist a request with no samples

Reply:
[requested change] log that the request does not already exist and so updates will not be persisted // a short blurb about how we do not persist requests that do not have any samples